README.md

```diff
@@ -73,6 +73,39 @@ Guide](https://solr.apache.org/guide/solr/latest/getting-started/solr-tutorial.h
 - Slack: Solr Community Channel.  Sign up at https://s.apache.org/solr-slack
 - IRC: `#solr` on [libera.chat](https://web.libera.chat/?channels=#solr)
 
+## Developer Documentation
+
+Solr uses Gradle for its build system. Here some useful hints to build and run Solr locally:
+
+- To build a Solr dev distribution for your local OS run:
+
+```
+./gradlew dev
+```
+
+- To run the dev distribution as Solr standalone:
+
+```
+cd ./solr/packaging/build/dev
+bin/solr start
+```
+
+- To run the dev distribution as SolrCloud:
+
+```
+cd ./solr/packaging/build/dev
+bin/solr start -c
+```
+
+- To list all the tasks you can execute:
+
+```
+./gradlew tasks
+```
+
+- Open a web browser and go to http://localhost:8983/solr/ to access the Solr Admin interface. You can also use the `bin/solr` script to create and manage Solr collections, or use the `bin/post` tool to index some sample data.
```

Comment:
please change bin/post to bin/solr post ;-)   bin/post is going away.

Reply:
ok, thanks for the review. I've updated the PR with this fix.